src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -18,6 +18,10 @@ public void testIntelDeflaterIsAvailable(){
             throw new SkipException("IntelDeflater not available on this platform");
         }
 
+        if ( SystemUtils.OS_ARCH.equals("ppc64le") ) {
```

Comment:
@droazen Good idea. Looked under utils/. Should we open a new file named NativeUtils.java, or we should piggy back on Utils.java?